.github/workflows/call-run-tests.yml

```diff
@@ -7,18 +7,19 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
 
-      - uses: actions/setup-node@v3
-        with:
-          node-version: 20
-          cache: 'npm'
+      - name: Setup test environment
+        uses: ./.github/actions/setup-env
+
+      # - name: Run tests in Chromium browser
+      #   run: npm run test:chrome
 
-      - name: Install dependencies
-        run: npm ci
+      # - name: Run tests in Firefox browser
+      #   run: npm run test:firefox
 
-      - name: Install Playwright Browsers
-        run: npx playwright install --with-deps
+      # - name: Run tests in Webkit browser
+      #   run: npm run test:webkit
```

Comment:
whats the difference between this and the new worflow above?

Reply:
the new workflow will run the tests by category with specific order... while the other one will run the tests either by browser or all tests without order